q2_emperor/_plot.py

```diff
@@ -21,8 +23,7 @@
 
 def generic_plot(output_dir: str, master: skbio.OrdinationResults,
                  metadata: qiime2.Metadata,
-                 other_pcoa: skbio.OrdinationResults,
-                 plot_name: str,
```

Comment:
I don't think we want to remove `plot_name` - this is part of the q2-emperor (not standalone emperor, though): https://github.com/qiime2/q2-emperor/blob/c0fe856ca72f8b625abd87b52bd2d6b1d4310299/q2_emperor/assets/index.html#L3